src/main/java/com/google/api/gax/core/PageAccessor.java

```diff
@@ -0,0 +1,21 @@
+package com.google.api.gax.core;
+
+/**
+ * Helper class that iterates the paged result while each page consists a group of entities.
```

Comment:
This is an interface, not a class... also it doesn't iterate anything, it is an Iterable. Suggestion: "Accessor for paged results from a list API method"